test/suites/dev/test-precompile/test-precompile-conviction-voting.ts

```diff
@@ -180,5 +199,167 @@ describeSuite({
         expect(revertReason).to.contain("Must be an integer between 0 and 6 included");
       },
     });
+
+    it({
+      id: "T06",
+      title: "should allow to vote split",
+      test: async function () {
+        const ayes = 1n * 10n ** 18n;
+        const nays = 2n * 10n ** 18n;
+        // Vote split
+        const rawTx = await createViemTransaction(context, {
+          to: PRECOMPILE_CONVICTION_VOTING_ADDRESS,
+          data: encodeFunctionData({
+            abi: convictionVotingAbi,
+            functionName: "voteSplit",
+            args: [proposalIndex, ayes, nays],
+          }),
+        });
+        const block = await context.createBlock(rawTx);
+
+        // Verifies the EVM Side
+        expectEVMResult(block.result!.events, "Succeed");
+        const { data } = expectSubstrateEvent(block, "evm", "Log");
+        const evmLog = decodeEventLog({
+          abi: convictionVotingAbi,
+          topics: data[0].topics.map((t) => t.toHex()) as any,
+          data: data[0].data.toHex(),
+        }) as any;
+
+        expect(evmLog.eventName, "Wrong event").to.equal("VoteSplit");
+        expect(evmLog.args.voter).to.equal(ALITH_ADDRESS);
+        expect(evmLog.args.pollIndex).to.equal(proposalIndex);
+        expect(evmLog.args.aye).to.equal(ayes);
+        expect(evmLog.args.nay).to.equal(nays);
+
+        // Verifies the Substrate side
+        // Since the vote is split, the total amount of votes is equal to %10
+        const referendum = await context
+          .polkadotJs()
+          .query.referenda.referendumInfoFor(proposalIndex);
+        expect(referendum.unwrap().asOngoing.tally.ayes.toBigInt()).to.equal(ayes / 10n);
```

Comment:
Should have a comment somewhere that says why you are dividing by 10 (in other words, mention how conviction multiplier of none is 0.1 X)

Reply:
There is one actually above. not enough?

Reply:
yes sorry